requirements.txt

```diff
@@ -1,32 +1,36 @@
-#
-# This file is autogenerated by pip-compile
-# To update, run:
-#
-#    pip-compile --output-file=requirements.txt requirements.in
-#
-asn1==2.2.0
-asn1crypto==0.24.0        # via cryptography
-attrs==18.1.0             # via pytest
-certifi==2018.11.29       # via requests
-cffi==1.11.5              # via cryptography
-chardet==3.0.4            # via requests
-click==6.6
-colorama==0.3.9           # via pytest

```

Comment:
Was this removal expected?

Reply:
No. No it was not. D:

I just ran the pip-compile instruction. Not sure what's caused that to be removed.

Reply:
Hmm, yes looks like `pytest` is the same version as before, so don't know why this would have been removed? The only thing that's changed is `pip-compile` version, so maybe it could be more intelligent into what it considers a transitive dependency to be. In any case, I don't think it's something to worry about, and we can leave as is.